docs/web-apps/automated-testing/cdp.md

```diff
@@ -53,10 +53,11 @@ public class SauceLabsTest {
         HashMap<String, Object> sauceOptions = new HashMap<>();
         sauceOptions.put("username", System.getenv("SAUCE_USERNAME"));
         sauceOptions.put("accessKey", System.getenv("SAUCE_ACCESS_KEY"));
-        sauceOptions.put("devTools", true);
+        sauceOptions.put("devTools", true);  // Legacy
```

Comment:
Same here, would abstain from using "legacy" now

Reply:
or maybe even better, provide examples using `webSocketUrl` instead, and leave the `devTools` ones as an optional example